### PR TITLE
Don't allow participant list insertion if the chairman is not present

### PR DESCRIPTION
**Pull request by lagartoverde**

### CODE REVIEW

File: app/components/participation-list/modal.hbs [hunk 1]
```diff
@@ -5,6 +5,14 @@
         as |Modal|>
   <Modal.Body>
     <div class="au-o-flow">
+      {{#if this.error}}
```
Comment (abeforgit) [inline app/components/participation-list/modal.hbs:8]: I would put this error next to the "insert" button

atm when you are scrolled down, you don't see the error and it feels like the insert button just doesnt work

**Review by abeforgit — CHANGES_REQUESTED** [review]
Just a smal QoL comment, works well otherwise
OK to merge for me, moving the alert is optional

**Review by abeforgit — CHANGES_REQUESTED** [review]
come to think of it, wouldn't it be better to simply disable the present toggle if the mandatee = the chairman?

We have no backup flow for when they do set the chairman as absent, we simply don't allow them to proceed. So why even have the option in the first place?

**Comment by lagartoverde** [comment]
I have made the changes you suggested, also kept the message and moved the error just in case, or if we want to add more errors to it

**Review by elpoelma — APPROVED** [review]
Works as expected.

**Pull request merged**